### DISCUSSION TITLE
Idea: Track Designs with inexact/unavailvable pathways workaround

### DISCUSSION THREAD

**Discussion by Superjustinbros** [OpenRCT2/OpenRCT2#15685] [Feature requests] [answered]
While I was playing around with some track designs, I noticed that if you don't have any of the path types in the current park that were originally saved with the track design, it nukes all the scenery associated with the design.

When I was younger, I originally thought that in vanilla RCT2, if the park you're playing in doesn't have the necessary paths saved into a track design, it would replace them with the normally mandatory-for-a-scenario grey tarmac/blue queue as opposed to removing all the associated scenery as well.

**Comment by Krutonium** [comment]
It would indeed make more sense to remove scenery that you don't have, and swap paths, wouldn't it...

**Comment by Superjustinbros** [comment]
Yes, it would.

**Comment by Gymnasiast** [comment, marked as answer]
This has been recently fixed.
